Makefile

```diff
@@ -105,12 +105,31 @@ else
 endif
 
 .PHONY: down
-down: mem-down
+down: mem-down explorer-down
```

Comment:
At least for me, when I bring down the network I want to bring down _everything_. Bringing the explorer up every time is possible, but it doesn't seem always desirable and it would make it slower to bring up the network. I'll check if there's a race condition.

Reply:
There is a race condition, I'd forgotten. So yes, this shouldn't include both even if it's almost always desirable.